v1/ao/layer.go

```diff
@@ -347,7 +348,7 @@ func (s *span) Err(err error) {
 	if err == nil {
 		return
 	}
-	s.Error("error", err.Error())
+	s.Error(reflect.TypeOf(err).String(), err.Error())
```

Comment:
@jiwen624 this broke tests that are looking for "error" and while `ErrorClass` does imply this behavior, maybe I should leave it as is?

Reply:
I changed it back, but curious your thoughts!

Reply:
I prefer `reflect.TypeOf(err).String()`, hoping that it could get the detailed error class. There are already "error" in both `Label` and `Spec`. A third "error" here is redundant.